### DISCUSSION TITLE
10월 17일 주간회의

### DISCUSSION THREAD

**Discussion by gracefulBrown** [woowacourse/prolog#1069] [General]
현재 진행중인 작업에 대해서 공유할 내용이 있으면 코멘트로 공유해주세요~

**Comment by euijinkk** [comment]
## 로드맵 프론트엔드 (with @wonsss)

로드맵 페이지 서버 모킹 후, 기능 구현 80% 완료
- 사이드바 처리만 진행하면 됩니다.

로드맵 기능 admin 페이지는 모킹해서 개발할 소요는 없을 것 같아서 간단하게 구상만 해본 후, 서버 나온 후 개발 시작하려고 합니다.

**Comment by EastHShin** [comment]
[인가과정 문서화 issue](https://github.com/woowacourse/prolog/issues/1070)
- [인가과정.md](https://github.com/woowacourse/prolog/blob/docs/combine/docs/backend/%EC%9D%B8%EA%B0%80%EA%B3%BC%EC%A0%95.md)

[DB ERD문서화 issue](https://github.com/woowacourse/prolog/issues/1029)
- [ERD.md](https://github.com/woowacourse/prolog/blob/docs/combine/docs/backend/ERD.md)

**Reply by her0807** [reply]
## 인가과정 문서화
현재 AOP 로 인가 과정이 진행되고 있는데 왜 interceptor 가 아니라 , AOP 로 인가처리를 하고 있는지 .. 구현된 배경에 대한 설명을 해주셔도 좋을 것 같아요.

**Comment by Yboyu0u** [comment]
[문서화 테스트 개선](https://github.com/woowacourse/prolog/pull/1068)  
[스터디로그 임시저장](https://github.com/woowacourse/prolog/pull/1053)

**Comment by al-bur** [comment]
## 임시저장

- 임시저장 기능 구현완료하고, 현재 PR 제출
- 코드 리뷰 대기중

https://github.com/woowacourse/prolog/pull/1067

**Comment by her0807** [comment]
## 로드맵 엔티티 설계 및 구현 완료
- https://github.com/woowacourse/prolog/tree/feature/1056-loadmap
- 해당 브런치 기준으로
   - keyword api  (루키)
   - quiz api 작업 예정( 수달)

**Comment by leo0842** [comment]
[모니터링 문서화](https://github.com/leo0842/prolog/blob/docs/combine-be-configuration-and-monitoring/docs/backend/%EB%AA%A8%EB%8B%88%ED%84%B0%EB%A7%81.md)

[형상관리 문서화](https://github.com/leo0842/prolog/blob/docs/combine-be-configuration-and-monitoring/docs/backend/%ED%98%95%EC%83%81%EA%B4%80%EB%A6%AC.md)

**Reply by her0807** [reply]
### 모니터링 문서화
현재 모니터링이 구축되어 있지 않지만, 구축의 필요성을 왜 느꼈는지 시도했던 과정도 포함되면 좋을 것 같아요. 
그리고 구축하는 과정에서 기록해놨던 내용들 (슬로한테 있던 것 같은데) 그것도 추가하면 좋을 것 같아요.

### 형상관리 문서화
여기에 PR 에 리뷰어 매칭할 때 PR 보내는 사람이 리뷰어 요청 보내고 본인이 직접 리뷰 어필해야한다 이런 꿀팁 같은 것도 자세히 남겨주면 좋을 것 같아요
그리고 깃헙 브런치 전략 (배포 과정)도 헷갈려서 그 내용도 있으면 좋을 것 같네요 ☺️ @leo0842 

에덴 고생 하셨습니다

**Reply by her0807** [reply]
@pocojang

**Comment by kwannee** [comment]
[디자인 시스템](https://www.figma.com/file/15qD4XpMvnbBNDMJbzCjLn/Prolog?node-id=0%3A1)
[프론트엔드 문서화](https://github.com/woowacourse/prolog/tree/docs/combine/docs/frontend)
[왜 디자인 시스템을 해야하는가](https://fine-town-5b1.notion.site/43962e7c10754645b57ee857247c3530)

**Comment by verus-j** [comment]
* [댓글 개수 조회](https://github.com/woowacourse/prolog/pull/1062)
* [빌드 실패하는 테스트 수정](https://github.com/woowacourse/prolog/pull/1045)
* [flyway validate 문제 해결](https://github.com/woowacourse/prolog/pull/1061)
